api/app/tox.ini

```diff
@@ -38,6 +38,7 @@ known_first_party = signals,tests
 
 [coverage:run]
 branch = True
+data_file = /tmp/.coverage
```

Comment:
Just because I am curious why was this a problem on your machine?
Also is this something that will change in the future? I know that Bart Jeukendrup talked about coverage reports and badges in github for the PR's and branches.

Reply:
The pytest-cov needs a temp file. If unspecified, it will use the current location. Currently the docker container defines 'datapunt' as user to run the test. This user will not always have access to the current folder. OSX behaves differently, and it works. Linux however will issue permissions errors and sqlite errors. We need to explicitly specify the datafile in /tmp to avoid these errors in linux.

Reply:
We could consider a `tempfs` mount on the Docker/Linux side of things (https://docs.docker.com/storage/tmpfs/).

Reply:
We are already using /tmp/test